src/Element/NetsEasyPaymentElement.php

```diff
@@ -7,9 +7,9 @@
 /**
  * Provides an OS2forms payment element'.
  *
- * @FormElement("os2forms_payment")
+ * @FormElement("os2forms_nets_easy_payment")
```

Comment:
Use the full module name in the element key:
```suggestion
 * @FormElement("os2forms_payment_nets_easy_payment")
```